modules/preparing-an-inital-cluster-deployment-for-mce.adoc

```diff
@@ -0,0 +1,401 @@
+    - source: "registry.redhat.io/ubi8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/ubi8"
+    - source: "registry.redhat.io/multicluster-engine"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/multicluster-engine"
+    - source: "registry.redhat.io/rhel8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/rhel8"
+    - source: "registry.redhat.io/redhat"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/redhat"
+----
+
++
+Additionally, ensure your certificate is present in the `additionalTrustBundle` field of the `install-config.yaml`.
+
++
+.Example `install-config.yaml`
+[source,yaml]
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  . zzzzzzzzzzz
+  .  -----END CERTIFICATE-----
+----
+
++
+[IMPORTANT]
+====
+The `oc mirror` command  creates a folder called **oc-mirror-workspace** with several outputs.
+This includes the `imageContentSourcePolicy.yaml` file that identifies all the mirrors you need for {product-title}  and your selected Operators.
+====
+
+. Generate the cluster manifests by running the following command:
+
++
+[source,terminal]
+----
+$ openshift-install agent create cluster-manifests
+----
+This command updates the `cluster manifests` folder to include a `mirror` folder that contains your mirror configuration.
+
+. Create a sub-folder named  `openshift` in the `<manifests_directory>` folder. This sub-folder is used to store the extra manifests that will be applied during the installation to further customize the deployed cluster.
+
+. For MCE, create the following manifests and save in the `<manifests_directory>/openshift` folder:
+
++
+.Example `mce_namespace.yaml`
+
++
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    labels:
+      openshift.io/cluster-monitoring: "true"
+    name: multicluster-engine
+----
+
++
+.Example `mce_operatorgroup.yaml`
+
++
+[source,yaml]
+----
+  apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    name: multicluster-engine-operatorgroup
+    namespace: multicluster-engine
+  spec:
+    targetNamespaces:
+    - multicluster-engine
+----
+
++
+.Example `mce_subscription.yaml`
++
+[source,yaml]
+----
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: multicluster-engine
+    namespace: multicluster-engine
+  spec:
+    channel: "stable-2.1"
+    name: multicluster-engine
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+----
++
+[NOTE]
+====
+You can install a distributed unit (DU) at scale with the Red Hat Advanced Cluster Management (RHACM) using the assisted installer (AI). These distributed units must be enabled in the hub cluster.
+The AI service requires persistent volumes (PVs), which is installed through the {product-title} Local Storage Operator.
```

Comment:
Note: here we're just installing the OLM operators, later the PVs will be created (manually)